cloud/jenkins/pgo_operator_aws_openshift-4.groovy

```diff
@@ -0,0 +1,317 @@
+                    gcloud components update kubectl
+                    gcloud version
+
+                    curl -s https://get.helm.sh/helm-v3.2.3-linux-amd64.tar.gz \
+                        | sudo tar -C /usr/local/bin --strip-components 1 -zvxpf -
+
+                    curl -s -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OS_VERSION/openshift-client-linux-$OS_VERSION.tar.gz \
+                        | sudo tar -C /usr/local/bin --wildcards -zxvpf -
+                    curl -s -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OS_VERSION/openshift-install-linux-$OS_VERSION.tar.gz \
+                        | sudo tar -C /usr/local/bin  --wildcards -zxvpf -
+
+                    sudo sh -c "curl -s -L https://github.com/mikefarah/yq/releases/download/3.3.2/yq_linux_amd64 > /usr/local/bin/yq"
+                    sudo chmod +x /usr/local/bin/yq
+                '''
+
+            }
+        }
+        stage('Build docker image') {
+            steps {
+                git branch: 'master', url: 'https://github.com/Percona-Lab/jenkins-pipelines'
+                withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER'), file(credentialsId: 'cloud-secret-file', variable: 'CLOUD_SECRET_FILE'),file(credentialsId: 'cloud-minio-secret-file', variable: 'CLOUD_MINIO_SECRET_FILE')]) {
+                    sh '''
+                        sudo git reset --hard
+                        sudo git clean -xdf
+                        sudo rm -rf source
+                        ./cloud/local/checkout $GIT_REPO $GIT_BRANCH
+
+                        cp $CLOUD_SECRET_FILE ./source/e2e-tests/conf/cloud-secret.yml
+                        cp $CLOUD_MINIO_SECRET_FILE ./source/e2e-tests/conf/cloud-secret-minio-gw.yml
+
+                        if [ -n "${PGO_OPERATOR_IMAGE}" ]; then
+                            echo "SKIP: Build is not needed, PG operator image was set!"
+                        else
+                            cd ./source/
+                            sg docker -c "
+                                docker login -u '${USER}' -p '${PASS}'
+                                export IMAGE_URI_BASE=perconalab/percona-postgresql-operator:$GIT_BRANCH
+                                ./e2e-tests/build
+                                docker logout
+                            "
+                            sudo rm -rf ./build
+                        fi
+                    '''
+                }
+            }
+        }
+        stage('Create AWS Infrastructure') {
+            steps {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'openshift-cicd'], file(credentialsId: 'aws-openshift-41-key-pub', variable: 'AWS_NODES_KEY_PUB'), file(credentialsId: 'psmdb-openshift4-secret-file', variable: 'OPENSHIFT_CONF_FILE')]) {
```

Comment:
We can not use the this psmdb-openshift4-secret-file secret. We need to create new one for PG operator

Reply:
@cap1984  please use this one pgo-openshift4-secret-file

Reply:
done